ramls/mod-users-bl.raml

```diff
@@ -218,6 +219,42 @@ resourceTypes:
           body:
             text/plain:
               example: "Internal server error"
+  /login-with-expiry:
+    post:
+      description: |
+        Allow a new user to login and return two cookies, one containing the user's refresh token
+        and one containting an access token. Both tokens have an expiration. The expiration time
+        for each is contained in the composite user token expiration proerty.
+      is: [permissionsExpandable, includeable]
+      headers:
+        User-Agent:
+        X-Forwarded-For:
+      body:
+        application/json:
+          schema: loginCredentials
+      responses:
+        201:
+          # NOTE: Set-Cookie headers are set in BLUsersAPI.java using the raw response object
+          # rather than through RMB code-generation. This is because RMB code generation doesn't
+          # let us do multiple haders with the same value, which we need to do here.
```

Comment:
We have comments in the implementation too. With this comment I'm trying to help anyone who is coming to the RAML file and expecting to see the response headers for `Set-Cookie` specified there with 201 as they would normally be if we were using RMB code gen for these headers.

Reply:
Ok, thank you, I didn't realise this workaround permeated into the interface definition as well.

Does this comment get pulled into the generated documentation?

Reply:
No it does not. But it can be done with in-line content as Markdown:
```
      responses:
        201:
          description: |
            Show me in the `generated` doc:
            * With Markdown.
            * See [FOLIO-1833](https://issues.folio.org/browse/FOLIO-1833)
```

Or via extra file being Markdown content.

```
      responses:
        201:
          description: !include note-201.md
```

Reply:
@dcrossleyau Thank you, that's useful to know

Reply:
@dcrossleyau - Thanks for sharing how to have the comment included in the generated docs. I'll change it to markdown.

Reply:
@dcrossleyau - I added a description property to the 201 response. See what you think.

Reply:
Yes, i verified it locally using [api-doc](https://dev.folio.org/guides/api-doc/) -- nice.